# Generated from /home/pmukherj/indoor_bag.bag on 2013-09-12.
mag_zero_x: 0.080455
mag_zero_y: 0.018629
mag_zero_z: -1.139871
mag_zero_radius: 0.512635
